Documentation/dev-guide/local_cluster.md

```diff
@@ -1,90 +1,152 @@
-# Setup a local cluster
+# Set up a local cluster
 
-For testing and development deployments, the quickest and easiest way is to set up a local cluster. For a production deployment, refer to the [clustering][clustering] section.
+For testing and development deployments, the quickest and easiest way is to configure a local cluster. For a production deployment, refer to the [clustering][clustering] section.
 
 ## Local standalone cluster
 
-Deploying an etcd cluster as a standalone cluster is straightforward. Start it with just one command:
+### Starting a cluster
+
+Deploying an etcd cluster as a standalone cluster is straightforward.
+Start it with just one command:
```

Comment:
this should be on the same line as `Deploying...` to conform with https://github.com/coreos/docs/blob/master/STYLE.md#one-sentence-per-line-deprecated